doc/de/Account-Basics.md

```diff
@@ -4,71 +4,72 @@ Account - Basics
 * [Zur Startseite der Hilfe](help)
 
 
-**Registrierung**
+## Registrierung
 
-Nicht alle Friendica-Knoten bieten die Möglichkeit zur Registrierung. 
-Wenn die Registrierung möglich ist, wird ein "Registrieren"-Link unter dem Login-Feld auf der Startseite angezeigt, der zur Registrierungsseite führt. 
-Die Stärke unseres Netzwerks ist, dass die verschiedenen Knoten komplett kompatibel zueinander sind. 
-Wenn der Knoten, den Du besuchst, keine Registrierung anbietet, oder wenn Du glaubst, dass Dir eine andere Seite möglicherweise besser gefällt, dann kannst Du hier eine <a href="https://dir.friendica.social/servers">Liste von öffentlichen Servern (Knoten)</a> finden und den Knoten heraus suchen, der am Besten zu Deinen Anforderungen passt.
+Viele, aber nicht alle Friendica-Knoten (Server) bieten die Möglichkeit zur Registrierung an.
+Falls der Friendica-Knoten, den Du besuchst, keine Registrierung anbietet, oder Du glaubst, dass Dir ein anderer Knoten möglicherweise besser gefällt, dann findest Du hier eine <a href="https://dir.friendica.social/servers">Liste von öffentlichen Friendica-Knoten</a>, aus der Du Dir eine netten Knoten heraussuchen kannst.
```

Comment:
The usual link syntax in Markdow is `[text](url)`. Though HTML syntax works as well I think.